pkg/controller/importconfig/worker_default.go

```diff
@@ -71,6 +71,7 @@ func (w *defaultWorker) generateImportSecret(ctx context.Context, managedCluster
 			BootstrapKubeconfig:     base64.StdEncoding.EncodeToString(bootstrapKubeconfigData),
 			RegistrationImageName:   registrationImageName,
 			WorkImageName:           workImageName,
+			ImageName:               registrationOperatorImageName,
```

Comment:
hrm it is the same value right now, since we use operator binary to run singleton agent. It is possible in the future that we run a more customized binary rather than using operator. I think we should avoid the confusion here.

Reply:
Got it.